libparsec/crates/client/tests/unit/certif/add_shamir_recovery_brief_certificate.rs

```diff
@@ -0,0 +1,355 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use libparsec_tests_fixtures::prelude::*;
+
+use crate::certif::{
+    CertifAddCertificatesBatchError, InvalidCertificateError, MaybeRedactedSwitch,
+};
+
+use super::utils::certificates_ops_factory;
+
+#[parsec_test(testbed = "minimal")]
+async fn ok(env: &TestbedEnv) {
+    let env = env.customize(|builder| {
+        builder.new_user("bob");
+
+        builder.new_shamir_recovery(
+            "bob",
+            1,
+            [("alice".parse().unwrap(), 1.try_into().unwrap())],
+        );
+    });
+    let alice = env.local_device("alice@dev1");
+    let ops = certificates_ops_factory(&env, &alice).await;
+
+    let mut shamir_recovery_certificates = env.get_shamir_recovery_certificates_signed();
+    shamir_recovery_certificates.pop();
```

Comment:
Why do we need that action ?

Reply:
Because the env builder creates `2` certificates (brief and recovery), do I need to comment ?

Reply:
Because I test the brief and not the recovery, and yes slicing is ok too